src/helpers/product-helper.php

```diff
@@ -10,16 +10,25 @@
 class Product_Helper {
 
 	/**
-	 * Get the product name in the head section.
+	 * Gets the product name.
 	 *
 	 * @return string
 	 */
-	public function get_name() {
+	public function get_product_name() {
 		if ( $this->is_premium() ) {
-			return 'Yoast SEO Premium plugin';
+			return 'Yoast SEO Premium';
 		}
 
-		return 'Yoast SEO plugin';
+		return 'Yoast SEO';
+	}
+
+	/**
+	 * Gets the product name in the head section.
+	 *
+	 * @return string
+	 */
+	public function get_name() {
+		return $this->get_product_name() . ' plugin';
 	}
 
 	/**
```

Comment:
I would add some unit tests for these two methods.
The more coverage we have, the better.

Reply:
Done